ftag/flavour.py

```diff
@@ -22,14 +22,11 @@ class Flavour:
     cuts: Cuts
     colour: str
     category: str
-    _px: str = None
+    _px: str = ""
```

Comment:
I think better to use `None` as the default, it's a bit safer

Reply:
I remember getting some warning from pre-commit when I was using `None` but let me try again

Reply:
yeah you need to update the type hint :)